en/docs/chapter_tree/binary_search_tree.md

```diff
@@ -116,9 +116,9 @@ Given a set of data, we consider using an array or a binary search tree for stor
 | Insert element | $O(1)$         | $O(\log n)$        |
 | Remove element | $O(n)$         | $O(\log n)$        |
 
-In ideal conditions, the binary search tree is "balanced," thus any node can be found within $\log n$ loops.
+Ideally, the binary search tree is "balanced", so that any node can be found within $\log n$ loops.
 
-However, continuously inserting and removing nodes in a binary search tree may lead to the binary tree degenerating into a chain list as shown in the figure below, at which point the time complexity of various operations also degrades to $O(n)$.
+However, if we keep inserting and removing nodes in a binary search tree, it may cause the binary tree to degenerate into a linked list as shown in the figure below, at which point the time complexity of various operations also degrades to $O(n)$.
```

Comment:
optional:
However, if we keep inserting and removing nodes in a binary search tree, it may cause the binary tree to degenerate into a linked list as shown in the figure below, at which point the time complexity of various operations also degrades to $O(n)$.
->
"However, if we continuously insert and remove nodes in a binary search tree, it may degenerate into a linked list as shown in the figure below, where the time complexity of various operations also degrades to $O(n)$."